docs/UserGuide.md

```diff
@@ -172,6 +172,42 @@ Format: `find KEYWORD [MORE_KEYWORDS]`
 Examples:
 * `find_add geylang` returns all users whose addresses contain `geylang`
 
+### Cloning a person : `clone`
+
+Clones a contact from FAPro at the specified index.
+
+Format: `clone INDEX`
+* Clones the person at the specified `INDEX`.
+* The index refers to the index number shown in the displayed person list.
+
+Examples:
+* `list` followed by `clone 2` clones the 2nd person in the address book.
+* `find Betsy` followed by `clone 1` clones the 1st person in the results of the `find` command.
+
+Acceptable parameters for INDEX:
+* Only accept **non-negative** int values that are less than the size of the address book. Cannot be 0. Index must be for a contact that has not already been cloned.
+
+![edit format](images/cloneformat.png)
+
+Precise expected outputs on success:
+* Message shown to the user: “Cloned Person: X”, where X are the details of the person who was cloned.
+* For example, if “Name: John Doe; Phone: 98765432; Email: johnd@example.com; Occupation: Barber; Address: Hougang Avenue 1; AppointmentDate: 2024-02-02; Tags: “ was cloned, the the output is “Cloned Person: John Doe; Phone: 98765432; Email: johnd@example.com; Occupation: Barber; Address: Hougang Avenue 1; AppointmentDate: 2024-02-02; Tags: ”
+
+![edit format](images/cloneresult.png)
+
+Precise expected outputs on failure: 
+* When no index, zero or a negative index is entered next to the clone command, the error message “Invalid command format! clone: Clones the person identified by the index number used in the displayed person list. Parameters: INDEX (must be a positive integer)” is returned to the user
```

Comment:
Missing a full stop at the end, minor issue! "...returned to the user."

Reply:
Perhaps can separate this into more lines, reflecting the screenshot of the GUI. 
Invalid command format!
clone: Clones...
Parameters: ...
etc

Reply:
Also for your parameters portion, I think it would be better to put a full stop here, and reflect this in the screenshot too. This can improve consistency :D "(must be a positive integer)."